tensorflow/java/src/main/java/org/tensorflow/Tensor.java

```diff
@@ -34,6 +41,352 @@
  * }</pre>
  */
 public final class Tensor implements AutoCloseable {
+
```

Comment:
Nit: Can you place all these specialized `create()` methods after the one that creates from an `Object` so that all the `create()` methods are grouped together.